cas-server-core-api-ticket/src/main/java/org/jasig/cas/ticket/registry/TicketRegistryState.java

```diff
@@ -15,7 +15,7 @@
      * @return Number of ticket-granting tickets in the registry at time of invocation
      *         or {@link Integer#MIN_VALUE} if unknown.
      */
-    int sessionCount();
+    long sessionCount();
```

Comment:
Thought about this some; You're right of course. the API change is not really warranted by the Java API semantics. It's done, to fix a bug as I explained with the inconsistent between the types of timeout values, and the call that attempts to return a count of tickets that operate based on that timeout value. So this is really a bug fix, that requires an API change. We have to pragmatic here to get around the bug for this intended release: Do we really think this is a breaking change in terms of what adopters would expect? Do we really expect folks to extend and override that behavior and use that (now breaking API) in extensions? I would argue that it's not, and it's not really an "extension point". The breaking change is warranted because it's low-impact as far as extensions go.